packages/xo-server-perf-alert/src/index.js

```diff
@@ -589,6 +589,7 @@ ${monitorBodies.join('\n')}`
 
       const entriesWithMissingStats = []
       for (const entry of snapshot) {
+        if (entry.object.physical_size <= 0) continue
```

Comment:
1. explicitly test the object type to avoid silencing other unrelated alerts
2. add a comment explain this line. If you were not the author and without the proper context, do you think you would be able to understand it?